rfhub2/version.py

```diff
@@ -1 +1 @@
-version = "0.29"
+version = "0.30"
```

Comment:
If you are going to merge it to #440 then there's probably no need to update version

Reply:
Thanks @pbylicki I wanted to keep the merge order and introduce smaller changes at once. After #440 would be merged, I will change base branch to master :)